### ISSUE TITLE
stats: wait until blockscout finishes indexing

### ISSUE THREAD

**Issue by sevenzing** [blockscout/blockscout-rs#683]
For now we have to manually wait until blockscout finishes indexing, and only after that start `stats` service (overwise statistic will be incomplete). This is super inconvenient, so
I suggest:

1. In `stats` determine if blockscout is currently in indexing status and just wait + return `{"status": "WAITING_FOR_BLOCKSCOUT_INDEXING"}` instead of `{"status": "SERVING"}` in `/health`
2. In frontend: show `Stats and charts` button in the menu, but in stats page go to /health endpoint and in case WAITING_FOR_BLOCKSCOUT_INDEXING show page like "We are currently indexing blockchain data. Please wait until blockscout finishes indexing."

**Comment by bragov4ik** [comment]
The waiting part is working; reporting extracted to a separate issue https://github.com/blockscout/blockscout-rs/issues/1123
